README.md

```diff
@@ -4,8 +4,113 @@
 [![CodeQL](https://github.com/znsio/teswiz/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/znsio/teswiz/actions/workflows/codeql-analysis.yml)
 
 # To Build
+
 `./gradlew clean build`
 
+# ![#f03c15](https://placehold.co/15x15/f03c15/f03c15.png) Breaking changes in Latest teswiz v0.0.83![#f03c15](https://placehold.co/15x15/f03c15/f03c15.png)
```

Comment:
This has been moved to the new docs/BreakingChanges-README.md file. We do not want to get it back here.

Reply:
updated